src/bridge/EspressoTEEVerifier.sol

```diff
@@ -14,50 +14,85 @@ import {
     ENCLAVE_REPORT_LENGTH
 } from "@automata-network/dcap-attestation/contracts/types/Constants.sol";
 import {EnclaveReport} from "@automata-network/dcap-attestation/contracts/types/V3Structs.sol";
+import {
+    V3QuoteVerifier
+} from "@automata-network/dcap-attestation/contracts/verifiers/V3QuoteVerifier.sol";
+import {BytesUtils} from "@automata-network/dcap-attestation/contracts/utils/BytesUtils.sol";
+import {Ownable} from "solady/auth/Ownable.sol";
+
 /**
  *
  * @title  Verifies quotes from the TEE and attests on-chain
  * @notice Contains the logic to verify a quote from the TEE and attest on-chain. It uses the V3QuoteVerifier contract
- *         to verify the quote. Along with some additional verification logic.
+ *         from automatato verify the quote. Along with some additional verification logic.
  */
+contract EspressoTEEVerifier is Ownable {
+    using BytesUtils for bytes;
 
-contract EspressoTEEVerifier is V3QuoteVerifier {
-    constructor(address _router) V3QuoteVerifier(_router) {}
+    // V3QuoteVerififer contract from automata to verify the quote
+    V3QuoteVerifier public quoteVerifier;
+    bytes32 public mrEnclave;
+    bytes32 public mrSigner;
 
-    /**
+    constructor(bytes32 _mrEnclave, bytes32 _mrSigner, address _quoteVerifier) {
```

Comment:
The reason I am accepting the `V3QuoteVerifier` contract here is because automata changed how they manage their contracts. Now each Verifier contract should be registered with the [PCCSRouter](https://github.com/automata-network/automata-dcap-attestation/blob/main/forge-test/AutomataDcapAttestationTest.t.sol#L55)